### DISCUSSION TITLE
ClassName Prop doesn't appear to be supported

### DISCUSSION THREAD

**Discussion by dylel** [gluestack/gluestack-ui#1698] [Feature Request]
### Description

I'm using nativeWind and trying to pass classNames into components and it seems to be swallowing the className as styles aren't applied to components

### gluestack-ui Version

1.011

### Platform

- [X] Expo
- [ ] React Native CLI
- [ ] Next
- [ ] Web
- [ ] Android
- [ ] iOS

**Comment by surajahmed** [comment]
@dylel Thanks for reporting this. We'll update you on this.
